.github/workflows/pages.yml

```diff
@@ -0,0 +1,27 @@
+name: Github Pages
+
+on: [push]
+
+permissions:
+  contents: write
+
+jobs:
+  build-github-pages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # repo checkout
+      - uses: actions-rs/toolchain@v1 # get rust toolchain for wasm
+        with:
+          toolchain: stable
+          target: wasm32-unknown-unknown
+          override: true
+      - name: Rust Cache # cache the rust build artefacts
+        uses: Swatinem/rust-cache@v1
+      - name: Downlaod and Install Trunk binary
+        run: wget -qO- https://github.com/thedodd/trunk/releases/latest/download/trunk-x86_64-unknown-linux-gnu.tar.gz | tar -xzf-
+      - name: Build # build
+        run: ./trunk build --release --public-url eframe_template # change this name to your project name
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
need to document that to enable docs, you first need to go to 
github repo -> settings -> pages -> source -> set to `gh-pages` branch + `/` root folder to deploy them.

might seem obvious, but not for noobs like me :) 

https://coderedart.github.io/eframe_template/ for live example